src/util/sendPaginatedMessage.ts

```diff
@@ -0,0 +1,79 @@
+import {
+	GuildMember,
+	MessageEmbed,
+	MessageReaction,
+	TextChannel,
+	User,
+} from 'discord.js';
+
+const emojis = {
+	back: '◀',
+	first: '⏮',
+	last: '⏭',
+	next: '▶',
+	stop: '⏹',
+};
+
+export async function sendPaginatedMessage(
+	embed: MessageEmbed,
+	pages: string[],
+	member: GuildMember,
+	channel: TextChannel,
+	timeout: number = 100000,
+) {
+	let curPage = 0;
+	const message = await channel.send(
+		embed
+			.setDescription(pages[curPage])
+			.setFooter(`Page ${curPage + 1} of ${pages.length}`),
+	);
+	if (pages.length === 1) return;
+
+	await message.react(emojis.first);
+	await message.react(emojis.back);
+	await message.react(emojis.stop);
+	await message.react(emojis.next);
+	await message.react(emojis.last);
+
+	const collector = message.createReactionCollector(
+		(reaction, user) =>
+			reaction.me &&
```

Comment:
Can you remove this like here? Checking for reaction.me is deprecated and is no longer needed in the version of discordjs that we use.

Reply:
Oh, sure